refactor: move copy to clipboard btn to code.js

rutwikhdev opened the pull request:
> no-docs

rutwikhdev committed:
refactor: move copy to clipboard btn to code.js

- ref: https://github.com/frappe/frappe/pull/26049

ankush commented:
The failing test is either related or something actually broke recently :new_moon_with_face:

rutwikhdev committed:
Merge branch 'frappe:develop' into minor-ui-refactor

rutwikhdev commented:
> The failing test is either related or something actually broke recently 🌚

Seems to be related fails while opening kanban settings locally as well. Let me take a look.

rutwikhdev committed:
test: fix cypress test

- ref: https://github.com/frappe/frappe/actions/runs/8762974274/job/24051318700?pr=26061
- show copy btn only when frm.doc is available
